_pages/user-experience/sign-in-sign-out.md

```diff
@@ -45,19 +45,19 @@ Don't use an icon instead of a text label. Buttons may include the Login.gov log
 
 <ul class="usa-icon-list padding-top-2">
   <li class="usa-icon-list__item">
-    {% include check_icon.html content=do_button_label style="text-green" %}
+    {% include icon.html icon_name="check_circle" content=do_button_label  style="text-green" %}
```

Comment:
Very nit: There's double spaces between `content=` and `style=` throughout this file, could they be collapsed?

Reply:
good catch, fixed in https://github.com/18F/identity-dev-docs/pull/449/commits/428cf11d811376546e6dfcae17f65c6ca7a84683